File: app/src/main/java/im/tox/antox/utils/Message.java
package im.tox.antox.utils;

import java.sql.Timestamp;

/**
 * Created by ollie on 10/03/14.
 */
public class Message {
    public int message_id;
    public int id;
    public String key;
    public String message;
    public boolean has_been_received;
    public boolean has_been_read;
    public boolean successfully_sent;
    public Timestamp timestamp;
    public int size;
    public int type;

    public Message() {
        super();
    }

    public Message(int id, int message_id, String k, String m, boolean received, boolean read, boolean sent, Timestamp time, int size, int type) {
        super();
        this.id = id;
        this.message_id = message_id;
        this.key = k;
        this.message = m;
        this.has_been_received = received;
        this.has_been_read = read;
        this.successfully_sent = sent;
        this.size = size;
        this.type = type;
        this.timestamp = time;
    }

}

File: app/src/main/scala/im/tox/antox/utils/Message.scala
package im.tox.antox.utils

import java.sql.Timestamp
//remove if not needed
import scala.collection.JavaConversions._

class Message(
  val id: Int,
  val message_id: Int,
  val key: String,
  val message: String,
  val has_been_received: Boolean,
  val has_been_read: Boolean,
  val successfully_sent: Boolean,
  val timestamp: Timestamp,
  val size: Int,
  val `type`: Int) {
}
